src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,114 @@
+.cards {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  padding: 50px 40px;
+
+  // @include catalog-card;
+
+  &__card {
+    display: flex;
+    flex-wrap: wrap;
+    align-items: center;
+    justify-content: center;
+
+    background: $white-color;
+    border: 1px solid #f3f3f3;
+    border-radius: 5px;
+
+    padding: 32px 16px 16px;
+    width: 200px;
+
+    &-page {
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      width: 160px;
+      height: 134px;
+      margin-bottom: 40px;
+    }
+
+    &-title {
+      font-weight: 500;
+      font-size: 12px;
+      line-height: 18px;
+      margin-bottom: 4px;
+      color: $bludark-color;
+    }
+
+    &-product-code {
+      width: $width-section;
```

Comment:
It's better to use a more descriptive variable name instead of '$width-section'. Consider renaming it to something like '$card-content-width'.